src/css/math.less

```diff
@@ -298,6 +298,21 @@
     }
   }
 
+  .mq-limit {
+    text-align: center;
+
+    span  {
+      display: block;
```

Comment:
oh, this is so that the `.mq-approaches` will be below the `lim`? How about instead of setting `display: block` on every span and then overriding that, just add `.mq-lim` class and select for that?